### PR TITLE
new-change

### PR DESCRIPTION
**Pull request by Rajitha-Kilaru**
new-change

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -52,4 +52,5 @@ app.post('/login', (req,res) => {
 
 app.listen(1234, ()=> {
     console.log('server is running... 3412')
+    console.log('server is running')
```
Comment (tirupathi2219) [inline server.js:55]: why we need two consoles

**Review by tirupathi2219 — COMMENTED** [review]
exeception

File: server.js (hunk 1 above)
Comment (Rajitha-Kilaru) [reply server.js:55]: just for checking collaborator access

Comment (tirupathi2219) [reply server.js:55]: ttttt

Comment (tirupathi2219) [reply server.js:55]: ghg

**Pull request merged**